nautobot_design_builder/context.py

```diff
@@ -298,10 +299,11 @@ class Context(_DictNode, LoggingMixin):
               or their native type.
     """
 
-    def __init__(self, data: dict = None, job_result: JobResult = None):
+    def __init__(self, data: dict = None, job_result: JobResult = None, design_name: str = ""):
```

Comment:
Context should be generic, practically just a dictionary. If the `design_name` is something that must be available to design jinja2 templates, then the `DesignJob` should add it when constructing the context.

Reply:
no, `design_name` is not (only) for jinja2 templates. The main goal is to be available in the helper functions of the context so I think it should be part of the Context class.